config/default.js

```diff
@@ -52,11 +67,14 @@ module.exports = {
   },
   ssltunnel: {
     enabled: true,
-    registration_endpoint: 'https://api.mozilla-iot.org:8443',
-    domain: 'mozilla-iot.org',
+    registration_endpoint: 'https://api.mozilla-iot-staging.org:8443',
```

Comment:
Please restore these config items to their original state.

Reply:
Done.